src/main/java/tictac7x/charges/items/U_ColossalPouch.java

```diff
@@ -0,0 +1,132 @@
+package tictac7x.charges.items;
+
+import static tictac7x.charges.store.ItemContainerId.INVENTORY;
+
+import com.google.gson.Gson;
+
+import net.runelite.api.Client;
+import net.runelite.api.ItemID;
+import net.runelite.api.Skill;
+import net.runelite.client.Notifier;
+import net.runelite.client.callback.ClientThread;
+import net.runelite.client.chat.ChatMessageManager;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.game.ItemManager;
+import net.runelite.client.ui.overlay.infobox.InfoBoxManager;
+import tictac7x.charges.ChargesImprovedConfig;
+import tictac7x.charges.item.ChargedItemWithStorage;
+import tictac7x.charges.item.storage.StorageItem;
+import tictac7x.charges.item.triggers.OnChatMessage;
+import tictac7x.charges.item.triggers.OnItemContainerChanged;
+import tictac7x.charges.item.triggers.OnStatChanged;
+import tictac7x.charges.item.triggers.TriggerBase;
+import tictac7x.charges.item.triggers.TriggerItem;
+import tictac7x.charges.store.ItemContainerId;
+import tictac7x.charges.store.Store;
+
+import static tictac7x.charges.ChargesImprovedPlugin.getNumberFromWordRepresentation;
+
+public class U_ColossalPouch extends ChargedItemWithStorage {
+    // TODO: save in config storage somehow
+    private int decayCount = 0;
```

Comment:
create new config key for example as:
```
@ConfigItem(
            keyName = colossal_pouch_decay_count,
            name = "Colossal pouch decay count",
            description = "Colossal pouch decay count",
        ) default int getColossalPouchDecayCount() { return 0; };
```

and then you can get it with `config.getColossalPouchDecayCount()`, and to save it you have to do `configs.setConfiguration(ChargesImprovedConfig.group, ChargesImprovedConfig.colossal_pouch_decay_count, 0);`.